src/lib/server/database/index.test.ts

```diff
@@ -126,6 +126,24 @@ it('should complete a full user journey', async () => {
         // TODO: add test case when the agent actually does have permission
         expect(await db.editTicketTitle(tid, 'New Title')).toStrictEqual(true);
 
+        // Creates a priority, then assigned agent sets ticket priority
+        const bytes = getRandomValues(new Uint8Array(21));
+        // TODO: add test case that checks if an agent is assigned to the ticket
+        const priority = Buffer.from(bytes).toString('base64');
+        const pid = await db.createPriority(priority, 0);
+        expect(pid).not.toStrictEqual(0);
+
+        const nonExistentPrio = 9696;
+        expect(await db.assignTicketPriority(tid, pid)).toStrictEqual(
+            db.AssignTicketPriorityResult.Success,
+        );
+        expect(await db.assignTicketPriority(tid, nonExistentPrio)).toStrictEqual(
+            db.AssignTicketPriorityResult.InvalidPriority,
+        );
+        expect(await db.assignTicketPriority(nonExistentTicket, pid)).toStrictEqual(
+            db.AssignTicketPriorityResult.TicketNotFound,
+        );
```

Comment:
For non-existent priority IDs, we can use `0` because Postgres doesn't give zeroed IDs.
```suggestion
        {
            const result = await db.assignTicketPriority(tid, pid);
            expect(result).toStrictEqual(db.AssignTicketPriorityResult.Success);
        }

        {
            const result = await db.assignTicketPriority(tid, 0);
            expect(result).toStrictEqual(db.AssignTicketPriorityResult.InvalidPriority);
        }

        {
            const result = await db.assignTicketPriority(nonExistentTicket, tid);
            expect(result).toStrictEqual(db.AssignTicketPriorityResult.TicketNotFound);
        }
```